alpha/apps/kaltura/modules/extwidget/actions/embedPlaykitJsAction.class.php

```diff
@@ -196,6 +203,8 @@ private function initMembers()
 		$this->bundleConfig = json_decode($confVars, true);
 		$this->setLatestOrBetaVersionNumber($confVars);
 		$this->setBundleName();
+		$namePrefix = substr($this->bundle_name, 0, 2);
+		$this->bundleWebDirPath = $this->bundleWebDirPath . $namePrefix . "/";
```

Comment:
please add "/" after $this->bundleWebDirPath
$this->bundleWebDirPath = $this->bundleWebDirPath . $namePrefix . "/"; should be $this->bundleWebDirPath = $this->bundleWebDirPath . "/" . $namePrefix . "/";

and if you want to make sure you dont have double "/" you can use rtrim to avoid this